client/state/analytics/reducer.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ANALYTICS_CONTINOUS_MONITOR_ON,
+	ANALYTICS_CONTINOUS_MONITOR_OFF
+} from 'state/action-types';
+import { createReducer } from 'state/utils';
+
+export const continuousTracking = createReducer( 'CHECKING', {
+	[ ANALYTICS_CONTINOUS_MONITOR_ON ]: () => 'TRACKING',
+	[ ANALYTICS_CONTINOUS_MONITOR_OFF ]: () => 'NOT_TRACKING'
+} );
+
+export default combineReducers( {
+	continuousTracking
+} );
```

Comment:
We're only defining one reducer here, so there shouldn't be any need to `combineReducers`:

```js
export default continuousTracking;
```